### PR TITLE
Event page

### PR DESCRIPTION
**Pull request by ofekashery**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@material-ui/icons": "^4.2.0",
     "@next/bundle-analyzer": "^8.1.1-canary.51",
     "@zeit/next-source-maps": "0.0.4-canary.1",
+    "classnames": "^2.2.6",
```
Comment (fangeugene) [inline package.json]: Use `clsx` instead.

**Comment by fangeugene** [comment]
Lint is going to fail, but I just wanted to push some of my updates.

File: src/components/StickySectionHeader/index.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ const useStyles = makeStyles(theme => ({
   }),
   label: {
     padding: `${theme.spacing(0.5)}px ${theme.spacing(2)}px`,
+    fontWeight: 400,
```
Comment (fangeugene) [inline src/components/StickySectionHeader/index.js]: Is there a specific reason for this? I was trying to keep the sticky headers consistent with the ones on the event list and team list pages.

Comment (ofekashery) [reply src/components/StickySectionHeader/index.js]: I think it looks better. What do you think?

**Comment by fangeugene** [comment]
Just going to merge this for now.

**Pull request merged**